src/PublicApiGenerator.Tool/Program.cs

```diff
@@ -250,7 +250,7 @@ public static void Main(string[] args)
         var asm = Assembly.LoadFile(fullPath);
         File.WriteAllText(outputPath, ApiGenerator.GeneratePublicApi(asm));
         var destinationFilePath = Path.Combine(outputDirectory, Path.GetFileName(outputPath));
-        if(File.Exists(destinationFilePath))
+        if (File.Exists(destinationFilePath))
```

Comment:
Not really related but I'm ok with the change. I'll probably align code styles anyway after merging